pyproject.toml

```diff
@@ -31,3 +31,6 @@ atomic = true
 multi_line_output = 3
 lines_after_imports = 2
 forced_separate = "pyrado"
+
+[tool.pyright]
+exclude = ["Pyrado/pyrado/environments/pysim/assets/*.egg"]
```

Comment:
May I ask you to remove this, since you also added it to #71

Reply:
Removed through force push